apps/cowswap-frontend/src/modules/account/containers/Transaction/styled.ts

```diff
@@ -383,10 +382,10 @@ export const TransactionInnerDetail = styled.div`
 
 export const TextAlert = styled.div<{ isPending: boolean; isExpired: boolean; isCancelled: boolean }>`
   background: ${({ theme, isPending }) =>
-    isPending ? transparentize(theme.attention, 0.85) : transparentize(theme.success, 0.85)};
+    isPending ? transparentize('#ff5722', 0.85) : transparentize(theme.success, 0.85)};
   margin: 6px 0 16px;
   padding: 8px 12px;
-  color: ${({ theme, isPending }) => (isPending ? theme.attention : theme.success)};
+  color: ${({ theme, isPending }) => (isPending ? '#ff5722' : theme.success)};
```

Comment:
Why the hardcoded values?

Reply:
The color is only used in this file, it shouldn't be in theme config.
I assume it must be replaced by some alternative from theme, but decided not to do it right now.
In the future we should make a search `/\#[\da-zA-Z]{3,6}/` and review all hardcoded values with Michel.